src/api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "/",
+  timeout: 5000,
+  headers: {
+    "Content-Type": "application/json",
+  },
+});
+
+export default api;
```

Comment:
Should we need 2 Axios instance file? if not please remove 1 of them in Upcoming pr

Reply:
it is needed since it gives different routes